exporter/honeycombexporter/config.go

```diff
@@ -41,5 +43,8 @@ type Config struct {
 }
 
 func (cfg *Config) Validate() error {
+	if err := cfg.QueueSettings.Validate(); err != nil {
+		return fmt.Errorf("queue settings has invalid configuration: %w", err)
```

Comment:
Maybe
```suggestion
		return fmt.Errorf("`sending_queue` settings has invalid configuration: %w", err)
```

to match the configuration name?